common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -122,7 +127,7 @@ public Random current() {
             logger.debug("-Dio.netty.noPreferDirect: {}", !DIRECT_BUFFER_PREFERRED);
         }
 
-        if (!hasUnsafe() && !isAndroid() && !IS_EXPLICIT_NO_UNSAFE) {
```

Comment:
Why was this removed?  It means that logging for this shows up even if no unsafe was explicitly set which was the point of adding this in the first place It means that with Netty 4.1.10.Final our users are back to seeing a message that they don't understand that makes them think their system is broken. Unless there is a good reason that this is a removed (I'm sorry if I'm missing it), I would like to open a PR to add it back.